src/spring-cloud/azext_spring_cloud/_utils.py

```diff
@@ -211,3 +211,12 @@ def _get_persistent_disk_size(tier):  # pylint: disable=too-many-return-statemen
     if tier == 'STANDARD':
         return 50
     return 50
+
+
+def get_portal_uri(cli_ctx):
+    """Get the Azure Portal URL in the current cloud."""
+    try:
+        return cli_ctx.cloud.endpoints.__getattribute__('portal_uri')
+    except Exception as e:
+        logger.debug("Could not get Azure Portal endpoint. Exception: %s", str(e))
+        return 'https://portal.azure.com'
```

Comment:
how this work in other public clouds like US, German etc

Reply:
I add a PR to add the `portal_uri` in Cloud, but seems there are something I missed in the PR. Will sync with  you guys in that thread. https://github.com/Azure/azure-cli/pull/15299